csireverseproxy/Makefile

```diff
@@ -47,7 +45,7 @@ docker-push: docker
 	make -f docker.mk push
 
 unit-test:
-	go test -v -coverprofile c.out -coverpkg $(coverpackages) ./...
+	go test -v -coverprofile c1.out ./...
```

Comment:
What's the reason for the change of variable name?

Reply:
No reason actually. probably saw c1.out in other places and just normalized it.